test/Templates.Test/Helpers/WebDriverFactory.cs

```diff
@@ -24,13 +26,19 @@ public static IWebDriver CreateWebDriver()
             var result = IsAppVeyor ? CreateFirefoxDriver() : CreateEdgeDriver();
             result.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1);
             return result;
+
+            IWebDriver CreateFirefoxDriver() =>
+                WebDriverFactory.CreateFirefoxDriver(new FirefoxOptions()
+                {
+                    AcceptInsecureCertificates = true
+                });
         }
 
         private static IWebDriver CreateEdgeDriver()
             => new EdgeDriver(EdgeDriverService.CreateDefaultService(BinDir));
 
-        private static IWebDriver CreateFirefoxDriver()
-            => new FirefoxDriver(FirefoxDriverService.CreateDefaultService(BinDir));
+        private static IWebDriver CreateFirefoxDriver(FirefoxOptions options)
+            => new FirefoxDriver(FirefoxDriverService.CreateDefaultService(BinDir), options, TimeSpan.FromSeconds(1));
```

Comment:
Can this change be reverted? It doesn't like it's used except by the unreachable code mentioned above.